code/client/src/core/dev_features.cpp

```diff
@@ -55,27 +54,27 @@ namespace MafiaMP::Core {
 
     void DevFeatures::Update() {
         if (_showEntityBrowser) {
-            _entityBrowser->Update();
+            _entityBrowser->Update(&_showEntityBrowser);
         }
 
         if (_showCameraStudio) {
             _cameraStudio->Update();
         }
 
-        if (_showVehicledebug) {
-            _vehicleDebug->Update();
+        if (_showPlayerDebug) {
+            _playerDebug->Update(&_showPlayerDebug);
         }
 
-        if (_showNetworkStats) {
-            _networkStats->Update();
+        if (_showVehicleDebug) {
+            _vehicleDebug->Update(&_showVehicleDebug);
         }
 
-        if (gApplication->_input->IsKeyPressed(FW_KEY_F7)) {
-            gApplication->GetImGUI()->ShowCursor(!_showCameraStudio);
-            MafiaMP::Game::Helpers::Controls::Lock(!_showCameraStudio);
-            _showCameraStudio = !_showCameraStudio;
+        if (_showNetworkStats) {
+            _networkStats->Update(&_showNetworkStats);
```

Comment:
Consistency over all codebase Update methods is required. Please avoid passing parameters there, and use a mutation method to toggle visibility state

Reply:
My knowledge of C++ ends here, I don't know how to do this :(

Can I have some help/example please?

Reply:
I'll suggest something

Reply:
What is your POV on this new approach?

https://github.com/MafiaHub/MafiaMP/pull/47/commits/90713d44e2662902f8ad9702b496ad2dd1984b82

Reply:
Does this correspond to the approach you recommend? https://github.com/MafiaHub/MafiaMP/pull/47/commits/ff7ef1d649cb59066246ba63c621bd3be929b90b

Reply:
Yes, but please make a getter `GetIsVisible() const` or `IsVisible() const` instead of raw member access.